### PR TITLE
Rename 'stored procedures' to 'user-defined functions' 

### LINKED ISSUE #147: Rename Stored Procedures to Stored Functions
PostgreSQL 11 supports `CREATE PROCEDURE` see https://www.postgresql.org/docs/11/static/xproc.html, change the terms to be more precise in the next version.

### PR DESCRIPTION
**Pull request by laurenceisla**
Closes #147

On stand-by until https://github.com/PostgREST/postgrest/issues/1210 is resolved.

### CODE REVIEW

File: releases/v7.0.0.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ Fixed
 Changed
 -------
 
-* :ref:`bulk_call` should now be done by specifying a ``Prefer: params=multiple-objects`` header. This fixes a performance regression when calling stored procedures.
+* :ref:`bulk_call` should now be done by specifying a ``Prefer: params=multiple-objects`` header. This fixes a performance regression when calling user-defined functions.
```
Comment (laurenceisla) [inline releases/v7.0.0.rst:66]: Not sure about changing this older changelog.

File: ecosystem.rst [hunk 2]
```diff
@@ -47,7 +47,7 @@ Example Apps
 External Notification
 ---------------------
 
-These are PostgreSQL bridges that propagate LISTEN/NOTIFY to external queues for further processing. This allows stored procedures to initiate actions outside the database such as sending emails.
+These are PostgreSQL bridges that propagate LISTEN/NOTIFY to external queues for further processing. This allows for user-designed functions to initiate actions outside the database such as sending emails.
```
Comment (laurenceisla) [inline ecosystem.rst:50]: Not sure if this references any stored procedure (like the ones used on triggers), or also to the user-defined functions exposed by PostgREST, or both.

File: releases/v7.0.0.rst (hunk 1 above)
Comment (steve-chavez) [reply releases/v7.0.0.rst:66]: Hm, no. Older changelogs shouldn't be changed.

File: ecosystem.rst (hunk 2 above)
Comment (steve-chavez) [reply ecosystem.rst:50]: Refers to internal functions - not exposed by PostgREST.

**Comment by steve-chavez** [comment]
The term [user-defined function](https://www.postgresql.org/docs/current/xfunc.html) does come from PostgreSQL docs but I don't think it's commonly used(I've seen UDF at some point though) - this might hurt searchability/SEO. Perhaps we can just call them "Functions"?

File: releases/v7.0.0.rst [hunk 3]
```diff
@@ -36,7 +36,7 @@ Added
 * Documentation improvements
 
   + Explanation for :ref:`Schema Structure <schema_isolation>`.
-  + Reference for :ref:`s_proc_embed`.
+  + Reference for :ref:`s_embedding_functions`.
```
Comment (laurenceisla) [inline releases/v7.0.0.rst:39]: Change to avoid broken link.

File: releases/v8.0.0.rst [hunk 4]
```diff
@@ -13,7 +13,7 @@ Added
 * Allow HTTP status override through the :ref:`response.status <guc_resp_status>` GUC.
   |br| -- `@steve-chavez <https://github.com/steve-chavez>`_
 
-* Allow :ref:`s_procs_variadic`.
+* Allow :ref:`s_functions_variadic`.
```
Comment (laurenceisla) [inline releases/v8.0.0.rst:16]: Ditto

**Review by laurenceisla — COMMENTED** [review]
I think these changelog changes are needed to avoid broken links.

**Comment by wolfgangwalther** [comment]
> On stand-by until https://github.com/PostgREST/postgrest/issues/1210 is resolved.

Since this issue is on-hold for now and the PR here is heavily outdated.. I will just close it. It would need a major rebase anyway, because the whole structure has changed a lot in the last three years. It's probably simpler to just redo it by now.

**Pull request closed without merging**